cli/src/main/kotlin/OrtMain.kt

```diff
@@ -77,6 +78,7 @@ class OrtMain : CliktCommand(name = ORT_NAME, epilog = "* denotes required optio
 
     private val logLevel by option(help = "Set the verbosity level of log output.").switch(
         "--info" to Level.INFO,
+        "--performance" to PERFORMANCE,
```

Comment:
Sorry for not noticing this easier, but this should probably also be exposed in the `Jenkinsfile`. I can do this in a follow-up commit, however.

Reply:
Done.